### PR TITLE
Added "to_f" and "to_i" methods to Time and TimeSpan

### PR DESCRIPTION
**Pull request by epitron**
...for Ruby compatibility.

### CODE REVIEW

File: src/time/time_span.cr [hunk 1]
```diff
@@ -104,6 +104,7 @@ struct TimeSpan
   def seconds
     (ticks % TicksPerMinute / TicksPerSecond).to_i32
   end
+  alias_method to_i, seconds
```
Comment (ysbaddaden) [inline src/time/time_span.cr]: I don't expect #to_i to return the seconds part of the range, but the total seconds minus milliseconds, something like `total_seconds.floor`

**Comment by asterite** [comment]
Thanks!

As @ysbaddaden says, I would make `TimesSpan#to_f` return `total_seconds` and `TimeSpan#to_i` to return `total_seconds.to_i`, or something like that.

I think `Time#to_f` is good.

But, please, can you include some specs in your pull request? It helps us review the code more quickly, and we can also be sure it works because it gets run on travis.

**Comment by epitron** [comment]
Hah, whoops! Sorry about that. I'll fix it up.

**Comment by epitron** [comment]
Okay, fixed!

**Comment by asterite** [comment]
Thanks!!

**Comment by epitron** [comment]
You're welcome!

**Comment by blocknotes** [comment]
When will this commit be available on a release...?
I see that it's on the master but not in the latest release

**Comment by asterite** [comment]
@blocknotes It's there on `Time::Span`. On `Time` use `epoch` and `epoch_f`

**Comment by blocknotes** [comment]
Ok, thank you @asterite

**Pull request merged**